src/keylessBackup/SignInWithEmail.test.tsx

```diff
@@ -51,7 +54,8 @@ describe('SignInWithEmail', () => {
     jest.clearAllMocks()
     store.clearActions()
     mockAuthorize.mockResolvedValue(undefined)
-    mockGetCredentials.mockResolvedValue({ idToken: 'google-token' })
+    mockGetCredentials.mockResolvedValue({ idToken: 'mock-token' })
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
nit: include gate name so we can come back to this for cleanup if / when we remove the gate